modmesh/app/sample_mesh.py

```diff
@@ -83,6 +83,25 @@ def help_other(set_command=False):
         view.mgr.pycon.command = cmd.strip()
 
 
+def help_mesh_viewer(path, set_command=False):
```

Comment:
I also added a sample command in sample_mesh app for, once mesh viewer app is completed, then this sample command can be removed.